src/pages/Home/components/Hero.styles.js

```diff
@@ -0,0 +1,43 @@
+import styled, { css } from 'styled-components';
+import { Link } from 'react-router-dom';
+
+export const HeroContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+export const Title = styled.h1`
+  ${({ theme }) => css`
+    font-family: ${theme.fonts.secondary};
```

Comment:
You can set the font-family etc inside the global styles for default text and headlines. They will be reused throughout the app, so it's way easier to define them globally instead of for each element